main.pm

```diff
@@ -498,7 +498,7 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
-        if (!get_var("OFW")) {
+        if (!get_var("OFW") && check_var("ARCH", "x86_64")) {
```

Comment:
The OFW var has become used throughout the codebase as effectivly equivalent to "ARCH like ppc64*" - given how we use the variable, are you sure your statement is correct?